tests/Receipts/ReceiptResponseTest.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+namespace Imdhemy\AppStore\Tests\Receipts;
+
+use Exception;
+use Imdhemy\AppStore\Receipts\ReceiptResponse;
+use Imdhemy\AppStore\ValueObjects\LatestReceiptInfo;
+use Imdhemy\AppStore\ValueObjects\PendingRenewal;
+use Imdhemy\AppStore\ValueObjects\Receipt;
+use Imdhemy\AppStore\ValueObjects\Status;
+use PHPUnit\Framework\TestCase;
+
+class ReceiptResponseTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function test_all_attributes_are_optional_except_status()
```

Comment:
afaik, @test annotation in PHPUnit is not needed if the test method starts with test*

Reply:
that's just my fetish for keeping things simple by stripping down redundancy

Reply:
I'm from the other team of explicitly set everything 😄

Reply:
your repo, so you make the rules 😉